### ISSUE TITLE
"new broken" filter

### ISSUE THREAD

**Issue by redsun82** [allure-framework/allure2#1240]
**Is your feature request related to a problem? Please describe.**
The "new failure" filter is quite convenient, it would be nice to have exactly the same way to quickly spot the tests that just got into the "broken" status.

**Describe the solution you'd like**
A "new broken" quick filter just like the "flaky" and "new failed" ones that would list the tests that are broken, but were passing in the previous run.

**Describe alternatives you've considered**
A workaround to have something similar would be in a specific project to force broken tests to appear as failed (for example, a pytest fixture turning all exceptions into assertion errors), so that new broken would appear alongside the new failures. One would however then lose the distinction between the two states.

**Additional context**
Nothing more to add.

**Comment by fludkov** [comment]
I completely agree with this feature request as I have a lot of faled and broken tests in my suite and I miss some new broken tests. Actually they mark as broken as selenium did not found some element on page (layout of page changed) and my tests did not throw AssertException on this but this is also I need to track. Please implement this filter
